src/cpp/r/session/graphics/RGraphicsUtils.cpp

```diff
@@ -40,7 +40,7 @@ namespace graphics {
 
 namespace {
 
-int s_compatibleEngineVersion = 8;
+int s_compatibleEngineVersion = 12;
```

Comment:
It looks like we set the compatible graphics engine version in a bunch of places, so we should sanity check that these changes make sense. (I'm a little confused because the versions declared in the various places are different)